jujugui/static/gui/src/app/components/usso-login-link/_usso-login-link.scss

```diff
@@ -0,0 +1,26 @@
+.usso-login {
+  position: relative;
+
+  &__notification {
+    background: #fff;
+    box-shadow: 0 1px 4.85px 0.15px rgba(0,0,0,0.2);
+    box-sizing: border-box;
+    color: #666;
+    font-size: 14px;
+    line-height: normal;
+    margin: 10px auto 0;
+    padding: 12px 25px;
+    position: absolute;
+    z-index: index($z-indexed-elements, usso-login-link);
+    top: 100%;
+    right: 10px;
+    transition: opacity 0.3s;
```

Comment:
It doesn't look like opacity is being set here. Let's remove the transition of were toggling display.